### ISSUE TITLE
Dragon Box Singles/Special Selection DVD support

### ISSUE THREAD

**Issue by Danfun64** [gravitypriest/dragon-radar#3]
For some people, the Dragon Box Singles are a lot easier to find than the actual Dragon Boxes. Their content isn't identical (slightly lower quality) but it's still better than the Blue and Green Bricks and the Double Features (can't say whether the DB Singles or the Funi Dragon Boxes look and sound better though)

**Comment by Danfun64** [comment]
...and speaking of singles...support for Funi GT (NEP dub and sub) and Z (pre-redub with horrible encoding xD)

**Comment by Danfun64** [comment]
...you said you were going to write a guide so we can do syncing by ourselves... when will that come out?

**Comment by gravitypriest** [comment]
FWIW I own some of the singles and the video is identical to the DBox, even the interlaced episodes.

As for the guide, I've begun it, but that is a serious time commitment and this is a hobby project.  Just haven't got around to it yet.

**Comment by Danfun64** [comment]
Video is identical... meaning the dvd layout is pretty much the same, like with dvd titles and stuff?

**Comment by Danfun64** [comment]
On another note, if you do support the Funi DBGT singles eventually, do you think you can set it up so that it combine stuff from both those and the Green Bricks (basically Green Bricks audio but with subtitles the NEP audio from the Funi DBGT singles)? I'm not sure if that can be done automatically, and the NEP audio for the dub doesn't exist with the Japanese music, but it will still be "complete".

**Comment by JTF195** [comment]
The same is true of the orange bricks and the funi dbox, which has subtitles for the NEPs. It would be nice to be able to combine them.

On the subject of GT though, I have the funi Hero's Legacy single dvd which uses the pre-remastered video, but neither it nor the green brick version have the original intro. Would it be possible to rip the DBGT_SP video as-is leaving the pre-OP intro intact?

**Comment by gravitypriest** [comment]
I am not sure whether the green brick and single have the same amount of frames, but the green brick retiming takes the 'special' intro into account.

**Comment by JTF195** [comment]
So I assume all that would need to change is to simply _not_ cut that part, and then start the english audio after the japanese-only intro has played?

Also,
>- Create straight DVD-to-MKV rips of a single release

MakeMKV can be used to dump VOB files straight into MKV containers. I'm not sure if it can be automated, but the main thing thing dragon radar would need to handle is naming the resulting files.

**Comment by gravitypriest** [comment]
Yeah no need to cut the intro off-- the -146 here: https://github.com/gravitypriest/dragon-radar/blob/master/params/offsets.json#L8336 is taking that intro into account when retiming.

**Comment by JTF195** [comment]
So what do I need to do to produce a final mkv without the intro cut? Is there some line of code in the source I can just uncomment?

**Comment by gravitypriest** [comment]
If you want the video without the intro, easiest way would be to create the movie normally, then use mkvtoolnix to split the file

**Comment by gravitypriest** [comment]
It may not cut cleanly due to I-frames, but at that point the only option is re-encoding but that's outside the scope of this project

**Comment by JTF195** [comment]
Just to clarify, what I'm talking about is the 30 second intro clip that plays before the opening credits on the R2 version . Dragon radar currently produces an mkv starting *at* the opening credits.

What do I need to do to include that (even though it doesn't have english audio or subs)

**Comment by gravitypriest** [comment]
Ah okay, sounds like you're talking about the next episode preview.  If you want that I recommend ripping it yourself using pgcdemux and merging it with the episode using mkvtoolnix.

**Comment by JTF195** [comment]
Yeah it's the NEP from the previous episode I'm guessing.
If you play the special from the DVD itself, it's included along with the episode.
Ripping the DVD with MakeMKV included it also.

Shouldn't it be fairly simple to modify dragon radar to include it?
(I think just change the starting VOB id, assuming I can figure out the right offset to use for the R1 start?)

**Comment by JTF195** [comment]
Alright. I demuxed the NEP by itself and calculated the total number of frames as 30.864 * (30/1.001), which rounds up to 925. Combined that with the existing 146 and to get an offset of -1071

Switched the demux param and it was smooth sailing:
```
			"R2": {
				"audio": ["jp"],
				"cells": null,
				"disc": "DBGT_SP",
				"pgc": 1,
				"type": "pgc",				
				"vid": [1, 2],
				"vts": 1
			}
```
It seems to have worked fine. It might be 1-2 frames off at the most, but I'm content.

**Comment by JTF195** [comment]
I got my bardock and trunks dvd today, and i noticed the bardock special also had a japanese-only NEP, so I used the same process to get an offset of -1113 for frame 0, and switched it to pgc 1 instead of vid 2.

All 3 specials also have a short closing title after the credits that is omitted unless demuxed as pgc 1.
